build/deploy/cockroach.sh

```diff
@@ -2,13 +2,13 @@
 
 set -eu
 
-if [ -f ./test.sh ]; then
-  ./test.sh
+if [ -f ../build/test.sh ]; then
+  ../build/test.sh
 fi
 
 if [ "${1:-}" = "shell" ]; then
   shift
-  /bin/bash "$@"
+  /bin/sh "$@"
```

Comment:
Oops, I was thinking of switching to use `busybox` as the base image, but reverted that. I'll revert this as well.